sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/ExpressionCompiler.java

```diff
@@ -1060,6 +1060,16 @@ node, new DBSPTypeBool(CalciteObject.EMPTY, false), DBSPOpcode.EQ,
                 return new DBSPApplyExpression(node, method, type, arg0, arg1);
 
             }
+            case ARRAY_REVERSE:
+            {
+                DBSPExpression arg0 = ops.get(0);
```

Comment:
Other functions check if the number of arguments matches.  Here you just ignore arguments 1-...
Same for sort_array in the other PR.

Reply:
Calcite handles that case for us. 

```sql
(no input file): Error in SQL statement
(no input file):1:26: error: Error in SQL statement: Invalid number of arguments to function 'ARRAY_REVERSE'. Was expecting 1 arguments
    1|CREATE VIEW VV AS SELECT array_reverse(ARRAY [1, 2, 3], ARRAY [1, 2, 3, 5])
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

java.lang.RuntimeException: org.apache.calcite.runtime.CalciteContextException: From line 1, column 26 to line 1, column 75: Invalid number of arguments to function 'ARRAY_REVERSE'. Was expecting 1 arguments
```

Reply:
Calcite keeps changing. Defensive programming never hurts